Solution1/ConsoleApp/AutoMap/WeatherProfile.cs

```diff
@@ -0,0 +1,26 @@
+﻿using BusinessLayer.DTOs;
+using DataAccessLayer.Model;
+
+namespace ConsoleApp.AutoMap
+{
+    public class WeatherProfile : AutoMapper.Profile
+    {
+        public WeatherProfile()
+        {
+            CreateMap<WeatherShortDTO, WeatherDTO>()
+                .ForMember(dest => dest.Temp, conf => conf.MapFrom(src => src.Main.Temp))
+                .ForMember(dest => dest.Comment, conf => conf.MapFrom(src => GetCommentByTemp(src.Main.Temp)));
+        }
+
+        private string GetCommentByTemp(double temp)
+        {
+            return temp switch
+            {
+                double n when n < 0 => "Dress warmly.",
```

Comment:
you can write just `_ when temp< 0 => "Dress warmly.",`

Reply:
+